platform-includes/distributed-tracing/custom-instrumentation/javascript.mdx

```diff
@@ -58,10 +58,12 @@ const baggage = document.querySelector("meta[name=baggage]")?.content;
 const pageLoadTransaction = Sentry.continueTrace(
   { sentryTrace, baggage },
   (transactionContext) => {
-    return Sentry.startTransaction({
+    Sentry.startSpan({
```

Comment:
m (but technically out of scope): Did `Sentry.continueTrace` ever return a transaction? I thought it only returned a `TransactionContext`?

Reply:
Either way, I think we should just remove the const here, no?

Reply:
ah, well it returns what is returned by the callback, but you're right, will remove this!